icubam/db/synchronizer.py

```diff
@@ -165,6 +165,15 @@ def sync_users_from_csv(self, csv_contents: TextIO, force_update=False):
     self.sync_users(users_df, force_update)
     return users_df.shape[0]
 
+  def sync_bedcounts_from_csv(self, csv_contents: TextIO, force_update=False):
```

Comment:
i am wondering if it would not be more appropriate to name those functions: `sync_icus`, etc. since we are already in the csvSynchronizer. And then the penultimate line becomes: `super().sync_users(...)`

Reply:
Will keep this explicit for now, I don't have a strong opinion but IIRC we agreed offline this was ok.